showcase/gapic-showcase/pom.xml

```diff
@@ -215,5 +215,144 @@
       <artifactId>opentelemetry-sdk-testing</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <!--  Logging testing dependencies  -->
+
   </dependencies>
+  <profiles>
+    <profile>
+      <id>slf4j2_logback</id>
+      <!--slf4j2, logback dependencies, and only compile logging test for 2.x-->
+      <activation>
+        <property>
+          <name>slf4j2_logback</name>
+        </property>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+          <version>2.0.16</version>
+          <scope>test</scope>
+        </dependency>
+        <dependency>
+          <groupId>ch.qos.logback</groupId>
+          <artifactId>logback-classic</artifactId>
+          <!--   Latest version with java8 support   -->
```

Comment:
does showcase run on java 8? Would it possible to use the a newer/ later version of logback to test with?

Reply:
Good point. My thought was that we would want to test for the oldest version and make sure it works. we test showcase on Java 11 onward. I will find newer version here.
Separately, this may fall into/be related to the project about dep upgrade strategy.

Reply:
upgrade done in https://github.com/googleapis/sdk-platform-java/pull/3403/commits/20017cc941f9d4fc92212f38c054a9a748e14787#diff-2786ec798ce7e963cf28c2b9ac79e3e7c7d0ac47114bfec84de71d09cfe93a20